release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -756,7 +756,7 @@ For AWS EBS and oVirt, this feature installs the CSI Driver Operator and driver
 If you installed a CSI Driver Operator and driver on an {product-title} 4.5 cluster:
 
 * The AWS EBS CSI Driver Operator and driver must be uninstalled before you update to a newer version of {product-title}.
-* The OpenStack Manila CSI Driver Operator is no longer available in Operator Lifecycle Manager (OLM). It has been automatically converted by the Cluster Version Operator.
+* The OpenStack Manila CSI Driver Operator is no longer available in Operator Lifecycle Manager (OLM). It has been automatically converted by the Cluster Version Operator. The original `openshift-manila-csi-driver-operator` namespace might still exist and must be deleted manually by a cluster administrator.
```

Comment:
Hi, I am thinking if using the **need** instead of **must** for the namespace deletion.

Reply:
I think it's best to leave it as is: we follow the IBM style guide, which recommends against using phrasal verbs like "need to" e.g. "need to be deleted by a cluster administrator"

Reply:
@jldohmann Hi, here I mean deleting the `openshift-manila-csi-driver-operator` namespace is not mandatory, it depends on the customer who installed the operator in 4.5, like if there is something others in this namespace. So I suggest to not use the "must", okay for me to use other words to match the IBM style guide.

Reply:
I see, thanks for clarifying. I will update the sentence to communicate this.